contents/blog/slack-closure.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2023-20-12
+title: "Why we're closing our public Slack community"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: CEO diaries
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+---
+
+One of the great things about PostHog being open source is that we’ve grown a vibrant community around the project. Since launch we’ve accepted code from over 500 contributors and swapped ideas with thousands of users in our public Slack group. 
+
+Now though, we’ve decided to close our public Slack group and focus efforts elsewhere. In this post I’ll explain why we’re doing this, and what comes next for the PostHog community. 
+
+> **Tl;dr:** We’re closing the Slack group on XX/XX/XX and inviting members to join us in the new [PostHog Community](/questions) instead. This decision only impacts the public Slack group, not the private Slack channels for users spending over $2,000 per month.
+
+## Why are we closing the PostHog Slack?
+
+PostHog has grown incredibly fast over the past four years and while that has ensured the public Slack group has remained a busy place, we think it’s arguably become _too_ busy. We often find users asking questions that have already been answered in previous chats, or re-asking unanswered questions if they drift too far back in the chat history. 
+
+This is a limitation of Slack itself. Message history disappears too quickly and search engines can’t index the chat, so everything which happens is too quickly lost.
+
+Over time we’ve tried a number of things to try and offset this, such as reorganizing channels or adding integrations to impose order. We even [built our own AI bot](/blog/aruba-hackathon#maxai-our-friendly-posthog-support-ai) which would answer questions automatically!
+
+We’ve also investigated moving to a paid plan, but the cheapest option is $7.25 USD per user per month. That’s a prohibitive cost for a channel with over 5,000 users and climbing, and would only increase the history to 30 days. 
+
+## Where will the community go?
```

Comment:
Worth sharing that actually /questions is pretty vibrant and has lots of engagement - it's not just starting from scratch? (e.g. X people are using and have answered Y questions etc.)

Reply:
Ha sorry, saw you put this later - but worth opening with it maybe